apps/web/src/pages/Pools/ViewPoolPage/ViewPoolPage.tsx

```diff
@@ -551,7 +551,7 @@ const ViewPoolPage = () => {
             onDuplicate={async ({ department }) => {
               return mutations.duplicate(
                 poolId,
-                data?.pool?.team?.id || "",
+                data?.pool?.team?.id ?? "",
```

Comment:
Pre-existing, but this mutation will fail anyway with an empty string team ID, right?